### ISSUE TITLE
meshlab snap package startup error with Qt Session management

### ISSUE THREAD

**Issue by jacknlliu** [cnr-isti-vclab/meshlab#159]
Startup the snap package, then I got the following error,
```
Qt: Session management error: None of the authentication protocols specified are supported
```
Nothing show up.

**Comment by thejeshgn** [comment]
+1 happening on Ubuntu

**Comment by pbackx** [comment]
Same thing for me on Ubuntu 17.10

    Gtk-Message: Failed to load module "gail"
    Gtk-Message: Failed to load module "atk-bridge"
    Gtk-Message: Failed to load module "canberra-gtk-module"
    Qt: Session management error: None of the authentication protocols specified are supported

Let me know if you need more information or if I can help with debugging.

**Comment by alemuntoni** [comment]
This issue should be solved in the latest version (2020.06). Feel free to reopen if the problem persists.
